scripts/tendermint/stop.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+LABEL_PART="tendermint/tendermint"
+
+CONTAINER_ID=$(docker container ls | grep -F "$LABEL_PART:" | awk '{print $1}')
```

Comment:
sure. this is a reliable way to find the PID

Reply:
The nice thing is, you can stop from a different console

Reply:
:+1: